naloge/algorithms_basics_test.go

```diff
@@ -564,4 +564,178 @@ func TestSumFrequentEvenDivByFour(t *testing.T) {
 			}
 		})
 	}
+}
+
+
+
+
+func TestShortestUniqueDigit(t *testing.T) {
+	tests := map[string]struct {
+		input       []string
+		expected    string
+		errExpected error
+	}{
+		"succes1": {
+			input:    []string{"123", "112", "789", "56"},
+			expected: "56",
+		},
+		"no repeating values1": {
```

Comment:
```suggestion
		"no repeating values": {
```

there shouldn't be numbers in test names